ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_default_is_type.schema.yml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  isis_default_is_type:
+    documentation_options:
+      filename: Fabric Variables
+      table: ISIS Settings
+    description: Additional underlay ISIS parameters
+    type: str
+    valid_values:
+      - level-1-2
+      - level-1
+      - level-2
+    default: "level-2"
```

Comment:
@carlbuchmann Can we double check if this is the true default. I see two different defaults in the docs.

Reply:
"level-2" seems to be correct for default.
```python
    @cached_property
    def is_type(self):
        if self.underlay_router is True:
            if self.underlay_routing_protocol in ["isis", "isis-ldp", "isis-sr", "isis-sr-ldp"]:
                default_is_type = get(self._hostvars, "isis_default_is_type", default="level-2")
                is_type = str(get(self._switch_data_combined, "is_type", default=default_is_type)).lower()
                if is_type not in ["level-1", "level-2", "level-1-2"]:
                    is_type = "level-2"
                return is_type
        return None

```